frontend/src/store/index.js

```diff
@@ -9,6 +9,7 @@ import { modules } from './modules';
 
 export const USER = 'user';
 export const UNSYNCRONISED_DRIVES = 'unsyncedDrives';
+export const UNSYNCRONISED_DRIVES_TOTAL_MILEAGE = 'unsyncedDrivesTotalMileage';
```

Comment:
both line 11 and 12 are wrong.. It should be `UNSYNCHRONISED`

Reply:
Sorry for that.